packages/calcite-components/src/utils/time.ts

```diff
@@ -167,6 +167,19 @@ export function getMeridiem(hour: string): Meridiem {
   return hourAsNumber >= 0 && hourAsNumber <= 11 ? "AM" : "PM";
 }
 
+export function getMeridiemOrder(locale: string): number {
+  const isRTLKind = locale === "ar" || locale === "he";
+  if (!isRTLKind) {
```

Comment:
Suggestion: for readability, consider having the shortest/simplest branch first.

```ts
if (isRTLKind) {
  return 0;
}

const timeParts = /*...*/
```

This enhances readability by avoiding the need to inverse the boolean name's meaning.